packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -104,6 +105,8 @@ import { shouldMakeEgg } from "../resources";
 import { lavaDogsAccessible, lavaDogsComplete } from "../resources/doghouse";
 import { hotTubAvailable } from "../resources/clanVIP";
 import { meatMood } from "../mood";
+import { freeFishyAvailable, shouldYachtzee } from "./yachtzee/lib";
```

Comment:
given the number of places these two functions are being used it might make more sense for them to just be in the regular lib